chia/rpc/wallet_rpc_api.py

```diff
@@ -61,7 +61,7 @@
 from chia.wallet.outer_puzzles import AssetType
 from chia.wallet.puzzle_drivers import PuzzleInfo, Solver
 from chia.wallet.puzzles.p2_delegated_puzzle_or_hidden_puzzle import puzzle_hash_for_synthetic_public_key
-from chia.wallet.singleton import create_fullpuz
+from chia.wallet.singleton import create_singleton_puzzle
```

Comment:
why is this better than `from chia.wallet import singleton` and then use `singleton.create_puzzle`?

Reply:
I think that that's a bad precedent to set unless you're importing something like a `Singleton` class.  Having a bunch of files all define `create_puzzle` seems like asking for confusing.

Reply:
It's not necessarily better. However, since the programmer using the singleton module can choose between
`import singleton` and `from singleton import foo`, you can have a lot of `foo` floating around.

And then you'll be in deep foo foo.